tensorflow_lite_support/tools/Build_TensorFlow_Lite_Support_Libraries_with_Bazel_and_Colab.ipynb

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    library = '//tensorflow_lite_support/java:tensorflowlite_support.aar' #@param ["//tensorflow_lite_support/java:tensorflowlite_support.aar", "//tensorflow_lite_support/java/src/java/org/tensorflow/lite/task/vision:task-library-vision", "//tensorflow_lite_support/java/src/java/org/tensorflow/lite/task/text:task-library-text", "//tensorflow_lite_support/java/src/java/org/tensorflow/lite/task/audio:task-library-audio","//tensorflow_lite_support/metadata/java:tensorflow-lite-support-metadata-lib"]
</pre><p>Better to make the comment more clear, such as one line per aar:</p><p>Support library: "//tensorflow_lite_support/java:tensorflowlite_support.aar"</p><p>Task Vision library: "//tensorflow_lite_support/java/src/java/org/tensorflow/lite/task/vision:task-library-vision"</p><p>Task Text library: "//tensorflow_lite_support/java/src/java/org/tensorflow/lite/task/text:task-library-text"</p><p>...</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tflite-support/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Configured so the user can see only "Support Library" and underneath it takes the value "//tensorflow_lite_support/java:tensorflowlite_support.aar"

Reply:
<p>What do you mean by "user can see only "Support Library""? Is "Support Library" an option? I couldn't find it anywhere in this colab.</p>

Reply:
Sorry for the misunderstanding. Check the newest commit.

Reply:
![lu](https://user-images.githubusercontent.com/26084498/127260409-f76bc190-36b1-4408-8074-364ae24f1040.JPG)

Reply:
<p>I see I see. It looks great! Thanks, George! </p>